sources/packages/api/src/services/ssh/ssh.service.ts

```diff
@@ -16,7 +16,7 @@ export class SshService {
   }
 
   /**
-   * Force the connection to be closed ensuring the an
+   * Force the dataSource to be closed ensuring the an
```

Comment:
Please rollback this change. The connection here means the SFTP connection, not the DB one.